packages/zero-runtime/README.md

```diff
@@ -18,6 +19,26 @@ A zero-runtime CSS-in-JS library that extracts the colocated styles to their own
   - [Color schemes](#color-schemes)
   - [Switching color schemes](#switching-color-schemes)
   - [TypeScript](#typescript)
+- [How-to guides](#how-to-guides)
+  - [Coming from Emotion or styled-components](#coming-from-emotion-or-styled-components)
+
+## Starter template
```

Comment:
```suggestion
## Starter templates
```

As we'll offer a Next.js _and_ Vite templates

Reply:
Also, maybe this Starter templates section should be within the "Getting started"?